app/src/features/mappings/MappingRow.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+
+import { Icon } from "@blueprintjs/core";
+import { IconNames } from "@blueprintjs/icons";
+import { makeStyles, TableCell, TableRow, Typography } from "@material-ui/core";
+import { ArrowForward } from "@material-ui/icons";
+import clsx from "clsx";
+
+import { useListResourceFilters } from "services/api/api";
+import { Resource } from "services/api/generated/api.generated";
+
+const useStyles = makeStyles((theme) => ({
+  row: {
+    cursor: "pointer",
+  },
+  cell: {
+    display: "flex",
+    alignItems: "center",
+  },
+  icon: {
+    width: 16,
+    height: 16,
+  },
+  flameIcon: {
+    fill: "#CC7831",
+    marginRight: theme.spacing(0.5),
+  },
+  tableIcon: {
+    fill: "#265EB1",
+    marginRight: theme.spacing(1),
+  },
+  text: {
+    margin: theme.spacing(0.5),
+  },
+  definitionId: {
+    fontWeight: 500,
+  },
+  inlineSpace: {
+    marginInline: theme.spacing(1),
+  },
+}));
+
+type MappingRowProps = {
+  mapping: Resource;
+};
+
+const MappingRow = ({ mapping }: MappingRowProps): JSX.Element => {
+  const classes = useStyles();
+  const { data: filters } = useListResourceFilters(mapping);
+  const filtersCount = filters?.length ?? 0;
+
+  return (
+    <TableRow hover className={classes.row}>
+      <TableCell className={classes.cell} size="small">
+        <Icon
+          icon={IconNames.TH}
+          className={clsx(classes.icon, classes.tableIcon)}
+        />
+        <Typography className={classes.text} color="textPrimary">
+          {mapping.primary_key_table}
+          {filtersCount > 0 && ` + ${filtersCount} filter(s)`}
```

Comment:
Can we add a condition on `filtersCount`, in order to separate the singular vs. plural case ?
**+ 1 filter** vs. **+ 4 filters**

It's not so costly but much better for the user in my opinion

Reply:
I handled it with the translation module